app.js

```diff
@@ -4,19 +4,21 @@ const express = require("express");
 const cors = require("cors");
 const morgan = require("morgan");
 const dotenv = require("dotenv");
+const routes = require("./routes");
```

Comment:
third-party 모듈과 커스터마이징 모듈을 구분해주기 위해 한칸 띄우는 것도 유지보수를 용이하게 하고 코드의 가독성을 높이는 하나의 컨벤션이 될 수 있습니다.
```suggestion

const routes = require("./routes");
```